modules/ROOT/pages/custom-application-alerts.adoc

```diff
@@ -37,39 +36,67 @@ This example illustrates:
 * How to send errors as notifications
 * How to send email alerts when notifications arrive
 
-You can create alerts in CloudHub with the software provided. If you want to create custom alerts, you need to get the https://www.mulesoft.com/exchange/#!/cloudhub-integration-connector[CloudHub Connector] and install it in Anypoint Studio.
-
-[TIP]
-For installation and configuration details, read the xref:3.8@mule-runtime::cloudhub-connector.adoc[CloudHub Connector User Guide].
-
 
 == Deploying Your Application
 
 To deploy your application, in the Runtime Manager click *Deploy Application* from the *Applications* menu. For more information, see xref:deploying-to-cloudhub.adoc[Deploying to CloudHub].
 
 == Triggering Your Notification
 
-Once your application is running, you can trigger the notification by simply going to your application's URL, for example: +http://YOURAPP.cloudhub.io/hello-notification+. The page should display a `Hello World` message, and automatically create a new notification in your Runtime Manager console. Next, return to the Runtime Manager console and click the new notification indicator bell indicator in the upper-right to show the alert message:
+Once your application is running, you can trigger the notification by going to your application's URL, for example: +http://YOURAPP.cloudhub.io/hello-notification+. The page displays a `Hello World` message, and automatically creates a new notification in your Runtime Manager console. 
+
+Return to the Runtime Manager console and click the new notification indicator bell in the upper-right to display the alert message:
 
 image::image2014-10-24-17-41-16-1.png[image2014-10-24+17-41-16-1]
 
 If the notification is sent after an exception, it attaches the
 `exception.message` and `exception.stacktrace` as custom properties of the notification, which are accessible from the Runtime Manager console.
 
-== Creating a Custom Alert
-
-Alerts allow you to send emails based on your application notifications. To create a custom alert for your application notification, click your application, and then click *Alerts* from the left menu.
-
-In the *New Alert* dialog:
+== Custom Alerts
 
-. Click *New alert*.
-. Enter an alert name: `Hello World`.
-. Optionally select your application name or leave the selection as *any*.
-. Select *custom application notification* as the event type.
-. Enter `Hello` as the word that triggers your notification. Recall the message you configured above is `Hello World`.
-. Enter email addresses that you wish to receive alert email.
-. Click *Create*.
+Alerts allow you to send emails based on your application's notifications. You can create standard alerts in CloudHub with the software provided. To create custom alerts, you need to get the https://www.mulesoft.com/exchange/#!/cloudhub-integration-connector[CloudHub Connector] and install it in Anypoint Studio.
```

Comment:
add tick marks & plus sign? or just tick marks? Is this a link we want to be active?